codehawk.properties

```diff
@@ -1,26 +1,14 @@
 category=External Analysers
 description=Analyze Java code smell
 homepageUrl=https://github.com/SDPMLab/CodeHawk
-archivedVersions=1.0
-publicVersions=1.0
+archivedVersions=1.2
```

Comment:
It can't be both public and archived. You'll (probably) _move_ 1.2 to this list once you release 1.3

Reply:
Removed value of archivedVersions